tidb-server/main.go

```diff
@@ -286,7 +286,9 @@ func registerStores() {
 	err := kvstore.Register("tikv", tikv.Driver{})
 	terror.MustNil(err)
 	tikv.NewGCHandlerFunc = gcworker.NewGCWorker
-	err = kvstore.Register("mocktikv", mockstore.MockDriver{})
+	err = kvstore.Register("mocktikv", mockstore.MockTiKVDriver{})
+	terror.MustNil(err)
+	err = kvstore.Register("unistore", mockstore.EmbedUnistoreDriver{})
```

Comment:
maybe we also need to add "unistore" to main.go:119, https://github.com/pingcap/tidb/pull/17156/files#diff-af2c9279ed41f7023f9b6b73cba2d18fR119